ci/jenkins/jobs/macros.yaml

```diff
@@ -293,3 +293,52 @@
           set -e
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test.sh --testcase '{conformance_type}' --registry ${{DOCKER_REGISTRY}} --testbed-type jumper
+
+- builder:
+    name: builder-capa-matrix
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          export JOB_NAME="matrix-${TEST_OS}-k8s-${K8S_VERSION//./-}-build-num"
+          ./ci/jenkins/test-vmc.sh --cluster-name "${JOB_NAME}-${BUILD_NUMBER}" --setup-only --provider aws --aws-region "us-west-2" --aws-access-key-id "${AWS_ACCESS_KEY}" --aws-secret-access-key "${AWS_SECRET_KEY}" --aws-service-user-name "${AWS_SERVICE_USER_NAME}" --aws-service-user-role "${AWS_SERVICE_USER_ROLE_ARN}" --aws-vpc-id "${CAPA_VPC}" --aws-subnet-id "${CAPA_SUBNET}" 
+          testcases=("e2e" "conformance" "networkpolicy")
+          for testcase in "${testcases[@]}"; do
+            ./ci/jenkins/test-vmc.sh --cluster-name "${JOB_NAME}-${BUILD_NUMBER}" --testcase "$testcase" --test-only --provider aws --aws-region "us-west-2" --aws-access-key-id "${AWS_ACCESS_KEY}" --aws-secret-access-key "${AWS_SECRET_KEY}" --aws-service-user-name "${AWS_SERVICE_USER_NAME}" --aws-service-user-role "${AWS_SERVICE_USER_ROLE_ARN}" --aws-vpc-id "${CAPA_VPC}" --aws-subnet-id "${CAPA_SUBNET}"
```

Comment:
prefer to structure properly for better readability with required arguments with multiple lines

./ci/jenkins/test-vmc.sh \
        --cluster-name "${JOB_NAME}-${BUILD_NUMBER}" \
        --testcase "$testcase" \
        --test-only \
        --provider aws \
        --aws-region "$AWS_REGION" \
        --aws-access-key-id "$AWS_ACCESS_KEY" \
        --aws-secret-access-key "$AWS_SECRET_KEY" \
        --aws-service-user-name "$AWS_SERVICE_USER_NAME" \
        --aws-service-user-role "$AWS_SERVICE_USER_ROLE_ARN" \
        --aws-vpc-id "$CAPA_VPC" \
        --aws-subnet-id "$CAPA_SUBNET"